packages/player/src/machines/video-machine.ts

```diff
@@ -295,7 +295,10 @@ export const videoMachine = createMachine<VideoStateContext, VideoEvent>({
             },
           },
         },
-        ended: {on: {PLAY: 'playing'}},
+        ended: {
+          entry: ['onVideoEnded'],
```

Comment:
we weren't doing anything when the playback ended (or even sending the event) but now it should run the action and be prepared to receive events appropriately